src/classes/AFFL_AccChange_TDTM.cls

```diff
@@ -80,11 +80,13 @@ public with sharing class AFFL_AccChange_TDTM extends TDTM_Runnable {
                        lookupFieldNameOld = AFFL_MultiRecordType_TDTM.afflMapper.contactLabelNames.get(lookupFieldLabelOld);
                    }
 
-                   if (isAfflTypeEnforced) {
-                       ERR_ExceptionHandler.handleAfflException(affl, lookupFieldName);
-                   } else if (lookupFieldName == null || lookupFieldNameOld == null) {
-                       return dmlWrapper;
-                   }
+                    if (isAfflTypeEnforced) {
+                        ERR_ExceptionHandler.handleAfflNullRecordTypeException(affl);
+                    } else {
```

Comment:
Why the change from `else if()`? I find the old way easier to read, but that's more of code styling question.

Reply:
Hey @lparrott, well IMO else if will get a little trickier if the conditional clause gets more complicated. Maybe we can have a standard for this thou.

Reply:
Good point, it's easier for debugging too, let's leave your updates as is.